lib/Cucumber.j

```diff
@@ -150,51 +150,68 @@ function simulate_double_click_on_point(x, y, flags)
     [cucumber_instance simulateDoubleClick:[x, y, flags]];
 }
 
-function simulate_dragged_click_view_to_view(aKey, aValue, aKey2, aValue2, flags)
```

Comment:
Even if drag and drop are simulated from cucapp now, it could be interested to have the possibility to simulate that from the CLI